submissions/smillims/html-css-popup/index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Last of Us</title>
+	<link rel="stylesheet" href="css/style.css">
+
+</head>
+
+<body>
+
+<div class="header">
+	<nav class="navbar">
+		<ul class="list">
+			<li class="list-item" tabindex="0">
+				<div class="menu">
+
+					<input class="menu-input input" type="checkbox" id="popup">
+					<label class="menu-label label" for="popup">
+						<img class="menu-img" src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAADIAAAAyCAYAAAAeP4ixAAAABmJLR0QA/wD/AP+gvaeTAAAAvklEQVRoge3WIQ7CQBCF4X8IYo9Qh+UItXUcAcvJsByhDiRHqMX1CHVFLWkW1Q3pkMn73MuaN8ludkBERETisGWY57kFbk5d1jqb2TOHfXGYgMO2faqlZdh5tfi1MIOUV2sALh5FKgzeBURE5GvXSkDj1GWt0cymHMoPsQXu2/ap1gGPHMKsKBrk34R57CIi4qL8Rxrg5NRlrd7MxhzKpfEIXLftU60DPoOEWVHCDFJerQl4eRSpoD1LREQkrDfOkR0gQ1QUHwAAAABJRU5ErkJggg==">
+					</label>
+					<div class="menu-inside">
+						<a class="menu-inside__link" href="#">
```

Comment:
If you want to show user a list of some entities it better to wrap them into `<ul>` | `<ol>` and `<li>`. 
It will be beneficial for [a11y](https://medium.com/web-standards/pragmatic-a11y-rules-b16dd2f77685)